src/containers/account/AccountContainer.js

```diff
@@ -40,10 +46,31 @@ const AccountContainer = () => {
   const dispatch = useDispatch();
 
   const getAtlasCredentialsRS :?RequestState = useRequestState([ACCOUNT, GET_ATLAS_CREDENTIALS]);
+  const regenerateAtlasCredentialsRS :?RequestState = useRequestState([ACCOUNT, REGENERATE_CREDENTIAL]);
   const atlasCredentials :Map = useSelector(selectAtlasCredentials());
 
+  const getAtlasCredentialsIsPending = isPending(getAtlasCredentialsRS);
+  const getAtlasCredentialsIsStandby = isStandby(getAtlasCredentialsRS);
+
+  const regeneratingAtlasCredentials = isPending(regenerateAtlasCredentialsRS);
+  const atlasCredentialsRegenerated = isSuccess(regenerateAtlasCredentialsRS);
```

Comment:
I think it's a good idea to handle the failure case. I don't have great ideas on how to design it, but one thought that comes to mind is to use a toast (similar to what we see in jira):

https://material-ui.com/components/snackbars/#customized-snackbars

Reply:
1000%